domain-models-maven-plugin/src/test/java/org/folio/rest/tools/ClientGeneratorTest.java

```diff
@@ -50,7 +50,8 @@ public void doesGenerateTestResourceClient() throws Exception {
 
 
   private String removeTrailingSpaces(String str) {
-    return TRAILING_SPACE_PATTERN.matcher(str).replaceAll("");
+    return TRAILING_SPACE_PATTERN.matcher(str).replaceAll("")
+        .replaceAll("\\r\\n", "\n"); // to revert ide automatic formatting
```

Comment:
```suggestion
        .replace("\r\n", "\n"); // to revert ide automatic formatting
```
No need to use the regexp method.

Reply:
I commited it.